content/en/cloudcraft/getting-started/connect-amazon-eks-cluster-with-cloudcraft.md

```diff
@@ -0,0 +1,105 @@
+---
+title: Connect an Amazon EKS cluster with Cloudcraft
+kind: guide
+---
+
+By scanning your Amazon EKS clusters, Cloudcraft allows you to visualize workloads and pods deployed inside to generate professional-looking system architecture diagrams.
+
+By using the existing [role-based access control (RBAC) authorization method provided by Kubernetes][1] and authorizing [our existing read-only IAM entity role][2], Cloudcraft requires no special software or agent to look inside your clusters.
+
+You can find more information on RBAC configuration and IAM entities in the link below:
+
+- [Managing users or IAM roles for your cluster][3]
+
+<section class="alert alert-info">
+  <p>The ability to scan Amazon EKS clusters and AWS accounts is only available to Cloudcraft Pro subscribers. Check out <a href="https://www.cloudcraft.co/pricing">our pricing page</a> for more information.</p>
+</section>
+
+## Prerequisites
+
+Before connecting your Amazon EKS clusters with Cloudcraft, you need to connect your AWS account and generate diagrams that include your clusters.
+
+Linked below, you will find documentation to help you connect your AWS account and familiarize yourself with Cloudcraft.
+
+- [Connect your AWS account with Cloudcraft][4]
+- [Create your first live AWS diagram][5]
+
+<p>You will also want to <a referrerpolicy="no-referrer" rel="noreferrer external help" href="https://docs.aws.amazon.com/eks/latest/userguide/install-kubectl.html">install and configure <code>kubectl</code></a>, a tool that lets you control Kubernetes clusters through the command line. AWS requires version 1.21 or higher, but we recommend using the latest version to avoid issues.</p>
```

Comment:
```suggestion
Before connecting your Amazon EKS clusters with Cloudcraft, you must connect your AWS account and generate diagrams that include your clusters.

To connect your AWS account and familiarize yourself with Cloudcraft, see the following articles:

- [Connect your AWS account with Cloudcraft][4]
- [Create your first live AWS diagram][5]

You should also [install and configure `kubectl`][7], a tool that allows you to control Kubernetes clusters through the command line. Cloudcraft recommends using the latest version to avoid issues.
```